### PR TITLE
Add send an issued token payment

### PR DESCRIPTION
**Pull request by jonathanlei**
Build an interactive tutorial on xrpl.org for how to send an issued (fungible) token to another account that trusts the same issuer, reference issues [here](https://github.com/XRPLF/xrpl-dev-portal/issues/1199)

### CODE REVIEW

File: open_bounties/0092 Interactive Tutorial for Sending An Issued Token Payment.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+## Overview
+
+### High Level Description
+Build an interactive tutorial on xrpl.org for how to send an issued (fungible) token to another account that trusts the same issuer, reference issues [here](https://github.com/XRPLF/xrpl-dev-portal/issues/1199)
+
+
+### requirements
+1. Write quality code samples in Javascript that shows steps to send an issued token payment, including: 
+    - [ ] include steps for troubleshooting common errors like tecPATH_DRY.
```
Comment (jonathanlei) [inline open_bounties/0092 Interactive Tutorial for Sending An Issued Token Payment.md:21]: might need help filling in more details here @mDuo13

File: open_bounties/0092 Interactive Tutorial for Sending An Issued Token Payment.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+## Milestones
+
+| # | High-Level Description | Details | Proposed Potential Award |
+| --- | --- | --- | --- |
+| 1 | Code Samples | Finish all the bullet points listed in requirement #1 - code samples  | $750|
+| 2 | Interactive tutorials | Make the code samples interactive | $750|
+
+
+## Helpful links/resources
```
Comment (jonathanlei) [inline open_bounties/0092 Interactive Tutorial for Sending An Issued Token Payment.md:34]: is there a corresponding reference doc on xrpl.org? @mDuo13

File: open_bounties/0092 Interactive Tutorial for Sending An Issued Token Payment.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+# Interactive Tutorial for Send an Issued Token Payment
+Bounty proposals require 3+ technical approvals _and_ budget approval before they are available to be worked on. A proposal PR with 3+ ✅s is not fully approved if it has not been merged. The bounty is only ready to be worked on after the PR has been merged.
+
+## Overview
+
+### High Level Description
+Build an interactive tutorial on xrpl.org for how to send an issued (fungible) token to another account that trusts the same issuer, reference issues [here](https://github.com/XRPLF/xrpl-dev-portal/issues/1199)
+
+
+### requirements
```
Comment (JST5000) [inline open_bounties/0092 Interactive Tutorial for Sending An Issued Token Payment.md:19]: nit: 
```suggestion
### Requirements
```

File: open_bounties/0092 Interactive Tutorial for Sending An Issued Token Payment.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+1. Write quality code samples in Javascript that shows steps to send an issued token payment, including:
```
Comment (JST5000) [inline open_bounties/0092 Interactive Tutorial for Sending An Issued Token Payment.md:20]:
```suggestion
1. Write quality code samples in Javascript that shows steps to send an issued token payment to another holder who trusts the same issuer, including: 
```

**Review by JST5000 — APPROVED** [review]
Seems solid after updating the wording.

**Review by khancode — APPROVED** [review]
LGTM after addressing all comments.

**Comment by intelliot** [comment]
marked as draft per https://github.com/XRPLBounties/Proposals/pull/91#issuecomment-1430452971

**Pull request closed without merging**